browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -110,7 +110,8 @@ class SlideShowPresenter {
 	_slideShowCanvas: HTMLCanvasElement = null;
 	_slideShowWindowProxy: ReturnType<typeof window.open> = null;
 	_windowCloseInterval: ReturnType<typeof setInterval> = null;
-	_currentSlide: number = 0;
+        _currentSlide: number = 0;
+    	_startSlide: number;
```

Comment:
FYI: `_currentSlide` will probably be removed; for known the current slide index ask `SlideShowNavigator`

Reply:
I have not used it yet. But I will keep on eye if changes